js/Game.js

```diff
@@ -20,7 +25,9 @@ class Game {
     let roundIndex = this.round - 1;
     let bonusRoundPuzzles = this.puzzleKeys[roundIndex - 1];
     let puzzleKeyIndex = this.puzzleKeys[roundIndex];
-    if (this.round === 5) {
+    if (this.round === 6) {
```

Comment:
I think I was running into issues where it would try to create a new bonus round after the bonus round was over. I don't think it affected functionality, cause it would still create a new game, but errors were popping up in the console because the lines after round === 5 were still running before the game got reset.